sources/packages/backend/apps/db-migrations/src/sql/Reports/Create-program-and-offering-status-report.sql

```diff
@@ -0,0 +1,120 @@
+INSERT INTO
+  sims.report_configs (report_name, report_sql)
+VALUES
+  (
+    'Program_And_Offering_Status_Report',
+    '(
+        select 
+          il.name as "Institution Location Name",
+          ep.program_name as "Program Name",
+          ep.sabc_code as "SABC Program Code",
+          ep.regulatory_body as "Regulatory Body",
+          ep.credential_type as "Credential Type",
+          ep.completion_years as "Program Length",
+          case
+            when (ep.delivered_on_site = true and ep.delivered_online = true) then ''Blended''
+            when ep.delivered_on_site = true then ''On-site''
+            when ep.delivered_online = true then ''Online''
+          end as "Delivery",
+          ep.course_load_calculation as "Credit or Hours",
+          ep.program_status as "Program Status",
+          case 
+            when ep.is_active = false then ''True''
+            when ep.is_active = true then ''False''
+          end as "Program Deactivated?",
```

Comment:
No need to convert here, we have been sending the raw values for boolean fields like this.
If a change is needed, the business will request and we can adapt it across all the files.